test_libs/pyspec/eth2spec/test/phase_0/block_processing/test_process_attestation.py

```diff
@@ -140,9 +140,50 @@ def test_old_source_epoch(spec, state):
 def test_wrong_shard(spec, state):
     attestation = get_valid_attestation(spec, state)
     state.slot += spec.MIN_ATTESTATION_INCLUSION_DELAY
+    sign_attestation(spec, state, attestation)
```

Comment:
Signature should be after modifying the `shard`

Reply:
Rebased it out, apparently had it right at first, then changed it. It's all good now.